javascript/react-js/react-testing-part-one.md

```diff
@@ -0,0 +1,177 @@
+### Introduction
+
+We've learned testing our vanilla JavaScript applications in the previous section. Testing is indeed a powerful tool that allows to write maintainable and flexible code. If you've followed along our lessons, we've been using the [Jest](https://jestjs.io/) framework. For React, we'll keep using Jest and add more capabilities to our tests using the React Testing library.
+
+Read the [introduction](https://testing-library.com/docs/react-testing-library/intro) page of the library before we get started.
```

Comment:
No particular reason, just thought it would be a good read to kick off the lesson

Reply:
I think that it would be better for people to just dive into the lesson, but if you feel fairly strongly that this article provides some good benefits before really digging into things I wouldn't call this a requirement for me to approve 🙂

Reply:
Considering that I don't have a strong reason, I'll go ahead and remove it